ChurchCRM/CRM
# Bugfix for CSV Import error

Modify SELECT LAST_INSERT_ID() to directly querying family id - and add logging

# Description & Issue number it closes 
Closes #6914 

## Screenshots (if appropriate)
N/A

## How to test the changes?
Use the `cypress/data/test_import.csv` and import as a new family.  Verify the head of household is associated with the same family as the other members.  Hopefully this will be testing properly after this change.

## Type of change


# How Has This Been Tested?

Tested via CSVImport from a clean/new install, using the above referenced test file.

# Checklist:
